ci_framework/roles/install_yamls/defaults/main.yml

```diff
@@ -30,8 +31,8 @@ cifmw_install_yamls_repo: "{{ cifmw_installyamls_repos | default(ansible_user_di
 cifmw_install_yamls_tasks_out: "{{ cifmw_install_yamls_out_dir }}/roles/install_yamls_makes/tasks"
 cifmw_install_yamls_whitelisted_vars:
   - DEPLOY_DIR
-  - METALLB_POOL
   - KUBECONFIG
+  - METALLB_POOL
```

Comment:
Any single new variable added here is a smell, and this one was no exception, used temporary but no one removed it :(

Reply:
:+1: